tests/test_docker_utils.py

```diff
@@ -184,27 +135,24 @@ def test_container_spec_no_input(self):
         self.assert_loads_eventually(info.url, 'Welcome to nginx!')
         self.assertTrue(self.client_wrapper.is_live(info.name))
 
-    def test_container_spec_with_input(self):
-        url = self.start_container({
-            'input': {'foo': 'bar'},
-            'container_input_path': '/usr/share/nginx/html/index.html'
-        }).url
-        self.assert_loads_eventually(url, '{"foo": "bar"}')
+    # TODO: We're no longer writing the input straight to disk:
```

Comment:
If you're getting to it soon, no worries but if not on immediate radar shift to an issue.

Reply:
Filed #196